aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -550,7 +549,8 @@ Tensor& _sparse_csr_mm_out(
     // TODO: replace with at::zeros when it's implemented for sparse csr
     zero = at::empty({mat1.size(0), mat2.size(1)}, mat2.options());
   } else {
-    zero = at::zeros({mat1.size(0), mat2.size(1)}, mat2.options());
+    zero = at::zeros(
+        {mat1.size(0), mat2.size(1)}, mat2.options().layout(result.layout()));
```

Comment:
We can just create an "empty" tensor to not rely on these `zeros`, no?

Reply:
yes: https://github.com/pytorch/pytorch/pull/85030